addons/binding/org.openhab.binding.amazonechocontrol/META-INF/MANIFEST.MF

```diff
@@ -9,16 +9,26 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
 Import-Package: com.google.gson,
+ javax.measure,
+ javax.measure.quantity,
  javax.servlet,
  javax.servlet.http,
  javax.ws.rs.core,
  org.apache.commons.lang,
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.jetty.io,
+ org.eclipse.jetty.server.session,
+ org.eclipse.jetty.util.component,
+ org.eclipse.jetty.util.ssl,
+ org.eclipse.jetty.websocket.api,
+ org.eclipse.jetty.websocket.client,
+ org.eclipse.jetty.websocket.common.scopes,

```

Comment:
I think you can fix the build by importing two additional packages:

```
 org.eclipse.jetty.client,
 org.eclipse.jetty.websocket.common,
```

Reply:
Hi wborn,
Thank's for your help. Your hint fixed my build.
My PR is feature complete, but I will do some code cleanup in the next days and wait for the beta tester results. 
Best regards,
Michael